library/core/Cargo.toml

```diff
@@ -24,8 +24,8 @@ path = "benches/lib.rs"
 test = true
 
 [dev-dependencies]
-rand = "0.7"
-rand_xorshift = "0.2"
+rand = { version = "0.8.5", default-features = false }
+rand_xorshift = { version = "0.3.0", default-features = false }
```

Comment:
Note to self: update https://github.com/bjorn3/rustc_codegen_cranelift/blob/master/patches/0022-sysroot-Disable-not-compiling-tests.patch